AndroidAnnotations/androidannotations-core/androidannotations-api/src/main/java/org/androidannotations/api/view/OnViewChangedListener2.java

```diff
@@ -0,0 +1,20 @@
+/**
+ * Copyright (C) 2016-2017 the AndroidAnnotations project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed To in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package org.androidannotations.api.view;
+
+public interface OnViewChangedListener2 {
```

Comment:
How could HasViewsBase look like? It would need to have a `findViewById` with a signature that is compatible to the other two versions. Or do i miss something in your suggestion?